src/settings.rs

```diff
@@ -229,6 +237,9 @@ impl Settings {
                 } else {
                     settings.dir = env::var("PWD").unwrap_or_else(|err| panic!(format!("McFly error: Unable to determine current directory ({})", err)));
                 }
+                if let Ok(results) = value_t!(search_matches.value_of("results"), u16) {
```

Comment:
If you want, you could fallback to a `MCFLY_RESULTS` env variable so that users can change the default easily.

Reply:
It seemed like a command-line switch fitted in more with existing code, as `results` only affects one subcommand, and the other MCFLY_ environment variables affect the program as a whole.

Reply:
I'd suggest we add both. I'm happy to do it.

Reply:
I've had another crack at it.  The behaviour now, is that 
'10' is in the code as a `Default`
`$MCFLY_RESULTS` over-rides this for general cases (e.g. user presses ctrl-r)
`--results` over-rides both for specific cases (user types out `mcfly search --results`)